### ISSUE TITLE
Suggestion box?

### ISSUE THREAD

**Issue by greg-drozd** [SwedbankPay/developer.swedbankpay.com#612]
Hi,

I cannot find any information in developer documentation about suggestions we (as developers) could have for your API. Please provide us with "suggestion box" feature :)

Thanks

**Comment by asbjornu** [comment]
Thanks for the great idea! We have this on our backlog. Meanwhile, you're free to post any suggestions you have here on GitHub.

**Comment by greg-drozd** [comment]
My suggestion is for api in general, not for docs. Should I use this repository for that?

**Comment by asbjornu** [comment]
Yes, I understood that. As we have no other means than sending off an e-mail to support or contact us through GitHub issues, I suggest you do the latter unless you prefer the former. :)
